manifests/generated/vmirs-resource.yaml

```diff
@@ -32,4 +32,9 @@ spec:
     - vmirss
     singular: virtualmachineinstancereplicaset
   scope: Namespaced
+  subresources:
```

Comment:
> This requires the CustomResourceSubresources feature gate to be enables for clusters prior to 1.11.

Is this CRD going to get rejected for 1.10 if the feature gate isn't enabled?

Reply:
No, will just strip the fields away from the CRD. I find that actually a little bit unpleasant in general but good in this case ...